addons/print/models/print_printer.py

```diff
@@ -32,10 +33,16 @@ class Printer(models.Model):
 
     name = fields.Char(string="Name", required=True)
     full_name = fields.Char(
-        string="Full Name", compute="_compute_full_name", store=True, 
+        string="Full Name",
+        compute="_compute_full_name",
+        store=True,
```

Comment:
Nit: Doesn't matter but I think black is splitting these out onto multiple lines because there is a comma after the last element, could you remove these and run black again? `barcode` and `queue` at least should be 1 line!